tests/trainer/flags/test_limit_batches.py

```diff
@@ -0,0 +1,75 @@
+# Copyright The PyTorch Lightning team.
```

Comment:
in this file and the other, from the set of tests you moved here, did you need to make any changes?

Reply:
yes.. just replaced the EvalModel with BoringModel. Nothing major.